cypress/integration/applitools/index.spec.ts

```diff
@@ -0,0 +1,31 @@
+// cy.wait is added to each one to allow page to fully load.
+// Potentially only an issue if running locally.
+
+describe('Docs site screenshots', () => {
+  it('screenshots home page', () => {
+    cy.visit('/');
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(2000);
+    cy.eyesOpen({
+      appName: 'paste_docs',
+      batchName: 'paste_docs_screenshots',
+      testName: 'Home',
+      browser: {width: 1440, height: 1020, name: 'chrome'},
+    });
+    cy.eyesCheckWindow('Home');
+    cy.eyesClose();
+  });
+
+  it('screenshots tokens page', () => {
+    cy.visit('/tokens');
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(2000);
+    cy.eyesOpen({
+      appName: 'paste_docs',
+      testName: 'Tokens',
+      browser: {width: 1440, height: 1020, name: 'chrome'},
+    });
+    cy.eyesCheckWindow('Tokens');
+    cy.eyesClose();
+  });
+});
```

Comment:
For now, drop the Applitools screenshots. I'm having some feels about Applitools right now.

Reply:
Got it. Agreed. Doesn't seem to work well with our set up.